src/Main.elm

```diff
@@ -384,75 +398,76 @@ gameItem model theme game =
 
         selectionIcon =
             if gameIsSelected then
-                inFront
-                    (row
-                        [ alignBottom
-                        , centerX
-                        , spacing 10
-                        , height <| px 56
-                        ]
-                        [ column [ width <| px 42, height <| px 42 ]
-                            [ image
-                                [ width <| fill
-                                , height <| fill
-                                , Border.color <| rgb255 255 255 255
-                                , Border.width 3
-                                , Border.rounded 45
-                                , clip
-                                ]
-                                { src = playerIcon, description = "" }
-                            ]
-                        , column
-                            [ width <| fillPortion 80
-                            , height <| px 42
-                            , paddingXY 20 2
+                row
```

Comment:
returning this element as Element Msg so we can have a clearer idea of what is used with `inFront` in the containing button